android/app/src/main/java/app/covidshield/MainApplication.java

```diff
@@ -65,6 +67,8 @@ public ReactNativeHost getReactNativeHost() {
     @Override
     public void onCreate() {
         instance = this;
+        Log.d("locale", Locale.getDefault().getLanguage());
```

Comment:
Now that we have an example on how we can use your generated files (https://github.com/cds-snc/covid-alert-app/commit/1585adb3fa667f5016e04e62fbf685f9eeb910d8) I think we could remove this.